interface/src/components/validation.js

```diff
@@ -1,13 +1,108 @@
 import React,{Component} from 'react'
-import { uploadSites } from './requests'
+import { withRouter, Redirect } from 'react-router-dom'
+import { uploadFiles } from './requests'
+import './stylesheets/validation.css'
 
 class Validation extends Component{
-    constructor(props){
-      super(props);
+  constructor(props) {
+    super(props) //since we are extending class Table so we have to use super in order to override Component class constructor
+    this.state = { //state is by default an object
+       onLoad: false,
```

Comment:
Perhaps rename this state variable to `hasLoaded` - I think the "onLoad" name is too close to an event's name

Reply:
Yeah, that sounds nice.